queries/latex/highlights.scm

```diff
@@ -0,0 +1,110 @@
+[
+  (generic_command_name)
+  "\\begin"
+  "\\end"
+  "\\newcommand"
+  "\\renewcommand"
+  "\\DeclareRobustCommand"
+] @function.macro
+
+(comment) @comment
+
+(bracket_group) @parameter
+
+[(math_operator) "="] @operator
+
+[
+  "\\usepackage"
+  "\\documentclass"
+  "\\input"
+  "\\subfile"
+  "\\subfileinclude"
+  "\\subfileinclude"
+  "\\includegraphics"
+  "\\addbibresource"
+  "\\bibliography"
+  "\\includesvg"
+  "\\includeinkscape"
+  "\\includeinkscape"
+  "\\usepgflibrary"
+  "\\usetikzlibrary"
+] @include
+
+[
+  "\\chapter"
+  "\\section"
+  "\\subsection"
+  "\\subsubsection"
+  "\\paragraph"
+  "\\subparagraph"
+] @type
+
+[
+(label_definition)
+(label_reference)
+(equation_label_reference)
+(label_number)
+] @label
+
+(key_val_pair
+  key: (_) @method ; like JS
+  value: (_))
+
+["(" ")" "[" "]" "{" "}"] @punctuation.bracket
+
+(chapter
+  ; command: (_) ;; not visible in grammar
+  text: (brace_group) @text.title)
+
+(section
+  ; command: (_) ;; not visible in grammar
+  text: (brace_group) @text.title)
+
+(subsection
+  ; command: (_) ;; not visible in grammar
+  text: (brace_group) @text.title)
+
+(subsubsection
+  ; command: (_) ;; not visible in grammar
+  text: (brace_group) @text.title)
+
+(paragraph
+  ; command: (_) ;; not visible in grammar
+  text: (brace_group) @text.title)
+
+(subparagraph
+  ; command: (_) ;; not visible in grammar
+  text: (brace_group) @text.title)
+
+;; Formatting
+
+((generic_command
+  name:(generic_command_name) @_name
+  arg: (_) @text.emphasis)
+ (#eq? @_name "\\emph"))
+
+((generic_command
+  name:(generic_command_name) @_name
+  arg: (_) @text.emphasis)
+ (#match? @_name "^(\\textit|\\mathit)$"))
+
+((generic_command
+  name:(generic_command_name) @_name
+  arg: (_) @text.strong)
+ (#match? @_name "^(\\textbf|\\mathbf)$"))
+
+(ERROR) @error
+
+;; Controversial
+; Add @text.bullet?
```

Comment:
I'm +1 on having this, maybe something more general `@text.adorment` or maybe both? in rst I'm using special for all of them https://github.com/nvim-treesitter/nvim-treesitter/blob/88ac3d23653a27973be8ff60e500848cacfcf968/queries/rst/highlights.scm#L10-L13